std-bits/base/src/main/java/org/enso/base/cache/DiskSpaceGetter.java

```diff
@@ -0,0 +1,14 @@
+package org.enso.base.cache;
+
+import java.io.File;
+import org.enso.base.CurrentEnsoProject;
+
+public class DiskSpaceGetter extends Mockable<Long> {
+  public DiskSpaceGetter() {
+    super(() -> getRootPath().getUsableSpace());
+  }
```

Comment:
Feel free to ignore as it's just personal style preference, but IMHO _here_ it would be more readable if `Mockable` just had a `protected abstract computeValue();` that would be overridden in a sub-class, instead of passing lambdas around.

Reply:
Done